### ISSUE TITLE
23.07.06(목) Issue

### ISSUE THREAD

**Issue by SaFD-00** [SaFD-00/2023_RCV_SUMMER_URP#4]
# <공부한 내용>
## 1. VGG16 성능 개선
- He Initialization
- Learning Rate Scheduler
- ckpt(https://tutorials.pytorch.kr/recipes/recipes/saving_and_loading_a_general_checkpoint.html)

## 2. VGG16 Baseline Code
- model에서 self.state_dict()
- view와 flatten 차이
